internal/core/data/v2/application/event.go

```diff
@@ -54,15 +66,11 @@ func AddEvent(e models.Event, ctx context.Context, dic *di.Container) (id string
 		))
 	}
 
-	//convert Event model to Event DTO
-	eventDTO := dtos.FromEventModelToDTO(e)
-	putEventOnQueue(eventDTO, ctx, dic) // Push event DTO to message bus for App Services to consume
-
-	return e.Id, nil
+	return nil
 }
 
-// Put event DTO on the message queue to be processed by the rules engine
-func putEventOnQueue(evt dtos.Event, ctx context.Context, dic *di.Container) {
+// PublishEvent publish incoming array of AddEventRequest through MessageClient
```

Comment:
Change to `PublishEvent publishes incoming AddEventRequest through MessageClient`

Reply:
Updated.